rs/replicated_state/src/replicated_state.rs

```diff
@@ -668,6 +668,18 @@ impl ReplicatedState {
         canisters_memory_usage + subnet_memory_usage
     }
 
+    /// Computes the memory taken by best-effort response messages.
+    pub fn best_effort_message_memory_taken(&self) -> NumBytes {
```

Comment:
Well I'm exactly asking because it's inconsistent :) Just between the different layers. I'd prefer if you could change it to something uniform everywhere, but not a blocker for me.

Reply:
This "taken" thing is the "public API" toward Execution (ignoring the fact that most of the replicated state is public anyway, the calls that Execution makes to collect this or that memory usage and the returned types all have "memory taken" in them).

That's what I meant by consistency. I agree that it's a sad state of affairs, but I'd rather not have to go through dozens of Execution modules and tests to rename things.